### PR TITLE
Added .env vars for mailer

### PR DESCRIPTION
**Pull request by serge1peshcoff**
Fixes MEMB-826

### CODE REVIEW

**Review by LeonVreling — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -57,6 +57,10 @@ PATH_OMS_SURVEY=oms-survey/docker/
 CORE_LOGIN=admin
 CORE_PASSWORD=5ecr3t
 
+SMTP_HOST=smtp.gmail.com
+SMTP_USER=admin@aegee.eu
```
Comment (linuxbandit) [inline .env.example]: can you change it to `admin@${BASE_URL}`?

**Review by linuxbandit — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (serge1peshcoff) [reply .env.example]: but it's the .env file, it won't work like that, we don't have envsubst here

Comment (linuxbandit) [reply .env.example]: you're right, then change it to `admin@DOMAIN` such that email crawlers don't get this address

**Review by linuxbandit — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (serge1peshcoff) [reply .env.example]: @linuxbandit one argument for force-push: if I'd do it with another commit, then it'll stay in the history so people/robots/crawlers would be able to access it

**Pull request merged**